### ISSUE TITLE
[Question]: UIE多卡训练报错问题

### ISSUE THREAD

**Issue by fuqiang-benz** [PaddlePaddle/PaddleNLP#4609]
### 请提出你的问题

python -m paddle.distributed.launch --nproc_per_node=3 --backend=gloo  ./finetune.py --device cpu --logging_steps 10 --save_steps 100 --eval_steps 100 --seed 1000 --output_dir ./checkpoint/model_best --per_device_train_batch_size 16 --per_device_eval_batch_size 16 --num_train_epochs 20 --learning_rate 1e-05 --do_train --do_eval --do_export --overwrite_output_dir  --disable_tqdm True --metric_for_best_model eval_f1 --load_best_model_at_end True --save_total_limit 1
